pkg/operator/controllers/subnets/subnet_serviceendpoint.go

```diff
@@ -19,11 +19,11 @@ import (
 func (r *reconcileManager) ensureSubnetServiceEndpoints(ctx context.Context, s subnet.Subnet) error {
 	// Do not add/reconcile service endpoints if egress lockdown is enabled.
 	if gatewayEnabled(r.instance) {
-		r.log.Info("Skipping service endpoint reconciliation since egress lockdown is enabled")
+		r.log.Debug("Skipping service endpoint reconciliation since egress lockdown is enabled")
```

Comment:
Why use `debug`?

Reply:
I imagine this will be the most common case, since this feature is enabled by default.